src/app/pages/client/explore/Server.tsx

```diff
@@ -468,6 +476,20 @@ export function PublicRooms() {
     explore({ instance: instanceId, since: undefined });
   };
 
+  const handleOpenMenu: MouseEventHandler<HTMLButtonElement> = (evt) => {
+    setMenuAnchor(evt.currentTarget.getBoundingClientRect());
+  };
```

Comment:
I think we should remove this menu from here and make save/bookmark button always visible on sidebar. because it is not much discoverable.